rpc/tests/main_test.go

```diff
@@ -38,7 +41,21 @@ func startServer(done chan struct{}) *exec.Cmd {
 		done <- struct{}{}
 	}()
 
-	time.Sleep(time.Second * 2) /// wait until gallactic starts
+	//Just for wait to ensure command executed and instance object is ready
+	addr := tConfig.GRPC.ListenAddress
```

Comment:
Please use `grpcBlockchainClient` to get blockchain client instance.

Reply:
fixed!

Reply:
@GheisMohammadi Thanks.